python/orca/src/bigdl/orca/learn/tf2/spark_runner.py

```diff
@@ -376,8 +377,16 @@ def validate(self, data_creator, batch_size=32, verbose=1, sample_weight=None,
 
         with self.strategy.scope():
             model = self.model_creator(self.config)
-            if self.model_weights:
-                model.set_weights(self.model_weights.value)
+
+        if self.model_weights is not None:
+            print("set weights from broadcast")
+            model.set_weights(self.model_weights.value)
+        elif self.model_dir:
+            start = time.time()
+            states = load_pkl(os.path.join(self.model_dir, self.application_id + "_states.pkl"))
```

Comment:
In the case of saving on local file system, it seems only the previous rank 0 node will have the model saved in `self.model_dir`

Reply:
For cluster mode, we need user to specify the shared fs path not local path, I will add this requirement later.